test/eden_test.exs

```diff
@@ -122,16 +122,6 @@ defmodule EdenTest do
     assert encode!(set) == "#\{:name, :age, \"John\", 42}"
   end
 
-  test "Encode Tag" do
```

Comment:
Because I got this failure:
```
test Encode Tag (EdenTest)
     test/eden_test.exs:125
     Assertion with == failed
     code: encode!(date) == "#inst \"1985-04-12T23:20:50.052Z\""
     lhs:  "#inst \"1985-04-12T23:20:50.520Z\""
     rhs:  "#inst \"1985-04-12T23:20:50.052Z\""
     stacktrace:
       test/eden_test.exs:127
```
which I now see is just a typo. I brushed it off as a timing. issue.